examples/discovery/discover.json

```diff
@@ -0,0 +1,4 @@
+{
+    "presentation_request_uri": "verifier.example.com/presentation_request",
+    "context": "register_kyc"
+}
```

Comment:
could these kind of small examples be in the text and not as a separate file..?

Reply:
it is inline, I just used to file to format the example - can remove it once we merge

Reply:
thank you